script/Ciel/Day 8/08_01_CIEL08_1B.txt

```diff
@@ -318,22 +347,25 @@ But―――
 [sha:85ca9a608321c3013872bcde278966b01ab860ab]{
 -- Page 60, Offset 18410.
 -- そこまで先輩を慕っていながら、俺はアイツの事も放っておけないでいる。
-Although I adore Senpai, I can't leave Arcueid alone.
+As much as I adore Senpai, I can't leave Arcueid alone.
```

Comment:
The whole thing with 放っておけない deserves a serious thought overall, because it's a much more charged statement than simply "can't leave her alone". It's something that I think I've expressed before, but the more I see it , the more it comes across as highly J>E

Of course, in this instance it also ties into the next sentence in a complicated manner so I really don't know a good way to deal with it. I might ask some other translator friends for thoughts.

Perhaps at the very least "I can't let Arcueid be" or some variation is a bit better.

Reply:
I don't think its really that J->E, if its massaged a bit to "I can't just leave Arcueid alone" it'll sound ok, and for the next line, it'd work better to go with "It's the vampires still in this city that I can't just let be". 

Using "let Arcueid be" is awkward and since the rest of the sentence makes it so obvious, I don't really think making them match 1:1 matters

Reply:
It's more about the weird-ass romantic implications that 放っておけない has. It doesn't come across as strongly in English as it does in Japanese.

https://machicon.jp/koigaku/column/127026/

Reply:
Yeah I also don't love direct tl of 放っておけない. Not being able to leave someone alone _does_ carry the same meaning in EN, but it's a weaker signal / can also parse as 'can't let her face stuff alone'. Thoughts on sth like
> As much as I adore Senpai, I still can't get Arcueid out of my head. / can't stop thinking about Arcueid.
> [...] It's the vampires still in this city that I can't get out of my head. / can't stop thinking about.

Reply:
That does work way better--I like it.
Do remember that it occurs with frightening frequency in the script though. Maybe its worth going back in translating the others ones in a similar vein.

Reply:
I reviewed the other instances, and honestly they all kinda seem fine as-is to me. Feel free to take a look as well though.